aries_cloudagent/resolver/default/peer3.py

```diff
@@ -64,7 +64,7 @@ async def _resolve(
         return doc
 
     async def create_and_store(self, profile: Profile, peer2: str):
-        """Injest did:peer:2 create did:peer:3 and store document."""
+        """Inject did:peer:2 create did:peer:3 and store document."""
```

Comment:
```suggestion
        """Ingest did:peer:2 create did:peer:3 and store document."""
```
This one might actually be meant to be "ingest," from the context; i.e. taking in a did:peer:2, creating a did:peer:3, and storing a document.

Reply:
Ingest was my first guess 😄 but then I doubted myself when I noticed that "inject" is also just 1 character away. Plus, inject is less gastrointestinal.

If ingest makes more sense, I'll amend that 👌

The other funny one was "encluding", which was the hardest to correct 😂 both "including" and "excluding" are just a 1 character diff. So that's another one I'd appreciate confirmation that I guessed right 😄

Reply:
I think it's probably clear enough as it is :slightly_smiling_face: The wording in general on this one is a bit odd